src/app/home-page/widget/news-updates/notification-updates.component.html

```diff
@@ -0,0 +1,18 @@
+<mat-card
+  *ngIf="(activeNotifications$ | async)?.length > 0"
+  class="alert alert-info mat-elevation-z mx-4 mt-4 mb-1"
+  data-cy="notification-updates-container"
+>
+  <div
+    *ngFor="let notification of activeNotifications$ | async"
+    fxLayout
+    fxLayoutAlign="space-between center"
+    class="pl-3"
+    data-cy="notification-updates-item"
+  >
+    <markdown class="markdown-remove-pmargin" [data]="notification.message"></markdown>
```

Comment:
Consider using `app-markdown-wrapper`, which sanitizes input. 

While we use that component intended for user-driven data, and it's **extremely** unlikely that a notification would have malicious content (somebody would have to get curator credentials to post a malicious notification, in which case we've got more serious problems), on the other hand, it's easy enough to use. And would slightly limit the damage if we were hacked.

Only commenting here, but everywhere else too.

Reply:
Makes sense :)